docs/use-events.md

```diff
@@ -10,8 +10,7 @@ The Event Catalog is the single source of truth for all events currently publish
 
 In the Event Catalog, you’ll find the following information for every event:
 
-* A brief overview of the event, including a description, domain, system of record, the topics it’s published on, expected throughput, the current lifecycle phase it’s in, and what other systems may be consuming it. 
-* Data elements: A summary of what each field in an event is.
+* A brief overview of the event, including a description, domain, system of record, the topics it’s published on, expected throughput, the current lifecycle phase it’s in, and what other systems may be consuming it.
```

Comment:
Could this be bullets?

Reply:
I agree with wanting bullets here. Also, when writing bulleted lists for web content, it helps to pay close attention to [parallel structure](https://webapps.towson.edu/ows/parallelism.htm) , hence:

  
**A brief overview of the event, including its:** 

*   description 
*   domain 
*   system of record
*   expected throughput
*   current lifecycle phase 
*   usage in topics
*   connections to consuming systems